sos/report/plugins/openvswitch.py

```diff
@@ -248,6 +254,8 @@ def setup(self):
                             # Not all ports are "bond"s, but all "bond"s are
                             # a single port
                             "ovs-appctl bond/show %s" % port,
+                            # In the case of IPSec, we should pull the config
+                            "ovs-vsctl get Interface %s options" % port,
```

Comment:
Is the capital `I` in "`Interface`" correct / allowed? From some googling, I see just "`ovs-vsctl get interface`" so no idea if the above is a valid command syntax or not..

Otherwise ACK to the PR.

Reply:
Yes, actually technically the table name is 'Interface' so best to use it, but ovs-vsctl will do a case-insensitive compare.

ex (from a system where I tested this change):
```
[root@wsfd-advnetlab09 openvswitch]# pwd
/var/tmp/sosreport-wsfd-advnetlab09-1111111111-2021-01-22-xtjhtic/sos_commands/openvswitch
[root@wsfd-advnetlab09 openvswitch]# cat ovs-vsctl_get_Interface_tun123_options 
{local_ip="10.19.107.21", remote_ip="10.19.188.117", remote_name=h2}
```

The actual json that gets returned from the ovsdb:
`2021-01-23T16:18:51Z|00020|jsonrpc|DBG|unix:/var/run/openvswitch/db.sock: received reply, result=[false,"00000000-0000-0000-0000-000000000000",{"Interface":{"3ccb9b7a-88f6-412d-9b6b-3b71386499b9":{"initial":{"name":"tun123","options":["map",[["local_ip","10.19.107.21"],["remote_ip","10.19.188.117"],["remote_name","h2"]]]}},"e1373bba-f11d-468d-860c-c6180aed90ee":{"initial":{"name":"br-ipsec"}}},"Open_vSwitch":{"44ce1914-0a85-43b6-81c1-3e338e0d309c":{"initial":{"cur_cfg":10}}}}], id=2`